index.html

```diff
@@ -175,5 +175,21 @@ <h3 class="view">Skills <img src="img/Normal Button/Tertiary/Icons/Union.png" al
         </div>
     </section> 
     <!-- End about me -->
+    <!-- Start contact -->
+    <footer>
+        <div class="contain">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xyyvwqrd" method="POST">
+                <input type="text" required name="full-name" placeholder="full name">
+                <input type="email" required name="Email" placeholder="email">
+                <textarea name="Message" placeholder="write your message here."></textarea>
```

Comment:
- [ ] As the previous reviewer requested, Kindly  an attribute `maxlength="30"` to your name input for validation
- [ ] Kindly add an attribute `maxlength="500"` to your message input for validation reasons

Reply:
I did them but i think I might forget to push them I will do it now